package.json

```diff
@@ -36,7 +36,8 @@
   "dependencies": {
     "compression": "^1.7.4",
     "express": "^4.17.1",
-    "net-browserify": "PrismarineJS/net-browserify"
+    "net-browserify": "PrismarineJS/net-browserify",
+    "sync-request": "^6.1.0"
```

Comment:
This should be a dev dependency, webpack put it in the bundle

Reply:
Sure i'll do that